src/Microsoft.ServiceFabric.AspNetCore.HttpSys/SR.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.ServiceFabric.Services.Communication.AspNetCore {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
```

Comment:
I guess this change is from the generated code not the handmade one.

Reply:
Yes, these are self generated changes